Sorting/MergeSort.php

```diff
@@ -8,49 +8,45 @@
  */
 function mergeSort(array $arr)
 {
-    if (count($arr) <= 1)
-    {
+    if (count($arr) <= 1) {
       return $arr;
     }
 
-    $mid = floor(count($arr) / 2);
-    $left = mergeSort(array_slice($arr, 0, $mid));
-    $right = mergeSort(array_slice($arr, $mid));
+    $mid = floor( count($arr) / 2 );
+    $leftArray = mergeSort( array_slice($arr, 0, $mid) );
+    $rightArray = mergeSort( array_slice($arr, $mid) );
```

Comment:
The PER style guide states that there should not be a space after the opening paren or before the closing paren in method and function calls

https://www.php-fig.org/per/coding-style/#47-method-and-function-calls

Reply:
I may be a little nitpicky on this.  I think it's fine, so I'll approve and merge